peer/peerstore_test.go

```diff
@@ -0,0 +1,253 @@
+package peer
```

Comment:
In general, I think its a good idea to move towards the `testify` package and use assert statements. They really help with debugging, giving useful explanations of the failure (especially helpful when other people try to make changes in your code!)